components/camera/rtsp/rtsp.go

```diff
@@ -162,21 +164,55 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 			}
 		}
 	}()
-	err = rc.client.Start(rc.u.Scheme, rc.u.Host)
+	// send Start to server URL and timeout if you can't
+	doneStart := make(chan struct{})
+	go func() {
```

Comment:
this has to use PanicCapturingGo in order to avoid a crash post panic.

Reply:
OK - removed the extra go-rountines. I think the problem may have just been in the test ... because I am getting timeout errors from the DESCRIBE and SETUP requests. I thought I might have needed to be worried about the client no timing out, but potentially that will not be a problem.